### ISSUE TITLE
hook up library flags w/ the server

### ISSUE THREAD

**Issue by Ligerx** [Ligerx/TachiWeb-React#9]
You can now get/set library sort/filter/display settings by making a GET/POST request to /api/v2/library/flags .

**Comment by Ligerx** [comment]
[API Link 1](https://github.com/TachiWeb/TachiWeb-Server/blob/master/TachiServer/src/main/java/xyz/nulldev/ts/api/v2/java/model/library/LibraryFlags.kt)
[API Link 2](https://github.com/TachiWeb/TachiWeb-Server/blob/master/TachiServer/src/main/java/xyz/nulldev/ts/api/v2/java/model/FilterStatus.kt)
[API Link 2](https://github.com/TachiWeb/TachiWeb-Server/blob/master/TachiServer/src/main/java/xyz/nulldev/ts/api/v2/java/model/SortDirection.kt)

**Comment by Ligerx** [comment]
Example Server GET Payload

```json
{
    "data": {
        "filters": [
            {
                "type": "DOWNLOADED",
                "status": "ANY"
            },
            {
                "type": "UNREAD",
                "status": "ANY"
            },
            {
                "type": "COMPLETED",
                "status": "ANY"
            }
        ],
        "sort": {
            "type": "ALPHA",
            "direction": "ASCENDING"
        },
        "display": "GRID",
        "show_download_badges": false
    },
    "success": true
}
```

**Comment by Ligerx** [comment]
Example POST payload. All fields must be included or the server will reject it.

```json
{
	"filters": [
        {
            "type": "DOWNLOADED",
            "status": "ANY"
        },
        {
            "type": "UNREAD",
            "status": "ANY"
        },
        {
            "type": "COMPLETED",
            "status": "ANY"
        }
    ],
    "sort": {
        "type": "ALPHA",
        "direction": "ASCENDING"
    },
    "display": "GRID",
    "show_download_badges": false
}
```

**Comment by Ligerx** [comment]
Implemented as of 3855c7c9ae3437df6c752c0e2c4118525e1f2eb8
